examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge/MCCastingPlayer.mm

```diff
@@ -58,13 +58,14 @@ - (void)verifyOrEstablishConnectionWithCompletionBlock:(void (^_Nonnull)(NSError
             cppDesiredEndpointFilter.productId = desiredEndpointFilter.productId;
         }
 
+        // TODO: In the following PRs. Removed desiredEndpointFilter to fix iOS app build issue. Replace desiredEndpointFilter with optional IdentificationDeclarationOptions. Add optional CommissionerDeclarationHandler callback parameter.
```

Comment:
Same comment as the JNI code. It should be easy to create an idOptions variable and set these vendor/productIds which are already being extracted from the request here than break the code.